hunts/src/hunt-view.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
```

Comment:
Isn't it a convention in React to name component files using upper camelcase to match their exported component name? e.g. `HuntViewMain.js` in this case

Reply:
yeah, I've always disliked camelcase in file names but i'll change it